atomicapp/external/templates/nulecule/answers.conf.sample.tpl

```diff
@@ -0,0 +1,7 @@
+[general]
+namespace = default
+provider = kubernetes
+
+[${app_name}]
+image = centos/httpd
+
```

Comment:
@dustymabe @cdrage The point was that ``atomicapp init`` should be dead simple and not do Nulecule introspection and other complicated stuffs.